superset-frontend/plugins/plugin-chart-table/src/TableChart.tsx

```diff
@@ -251,6 +253,13 @@ function SelectPageSize({
 const getNoResultsMessage = (filter: string) =>
   filter ? t('No matching records found') : t('No records found');
 
+const sanitizeHtmlValue = (value: DataRecordValue): DataRecordValue => {
```

Comment:
can't use the sanitizeHtml from the utils? Guessing this is different? does this function belongs in the utils or is it component-specific?